test/bot.test.js

```diff
@@ -42,7 +42,7 @@ describe('Staging Merger Bot', () => {
       const configHash = "{ enabled: true, 'label-name': 'On Staging', comment: true }"
       const encodedConfig = Buffer.from(configHash).toString('base64')
       nock('https://api.github.com')
-        .get('/repos/soberstadt/test-merge-repo/contents/.github/merge-bot.yml')
+        .get('/repos/soberstadt/test-merge-repo/contents/.github%2Fmerge-bot.yml')
```

Comment:
This seems like an odd change... But it's a test. If it's working 🤷‍♂️

Reply:
I agree. When my tests started failing I assumed it was broken somehow but the release notes confirm it:
> URL parameters are now always encoded when using context.github.* methods. For example, if you use context.github.repos.getContent( owner, repo, path ) make sure to not encode the value for path. Also if you were mocking http requests in your tests, replace e.g. repos/octocat/hello-world/contents/.github/config.yml with repos/octocat/hello-world/contents/.github%2Fconfig.yml